src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -217,6 +219,46 @@ export default {
 				this.open = false
 			}
 		},
+		afterShow() {
```

Comment:
Isn't it necessary to remove the event listener on `after-hide`?

Reply:
✅ i added it. (My thinking was, that it is not necessary, because the element is removed from the DOM. But i am not sure. :))